### PR TITLE
gRPC Stream class docs

### PR DESCRIPTION
**Pull request by olegbespalov**
## What?

This change fixes the missed gRPC Stream class signature and improves its documentation.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [x] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

Spotted during https://community.grafana.com/t/k6-server-grpc-streaming-with-accesstoken-metadata/117879

### CODE REVIEW

**Review by mstoykov — APPROVED** [review]

**Review by joanlopez — APPROVED** [review]

File: docs/sources/next/javascript-api/k6-net-grpc/_index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The `k6/net/grpc` module provides a [gRPC](https://grpc.io/) client for Remote P
...
 | [Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response)                                          | Returned by RPC requests.                                                                                                                               |
 | [Constants](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/constants)                                        | Define constants to distinguish between [gRPC Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response) statuses.         |
-| [Stream](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                                              | Creates a new GRPC stream.                                                                                                                              |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new GRPC stream.                                                                                                                              |
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-net-grpc/_index.md:21]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new gRPC stream.                                                                                                                              |
```

File: docs/sources/next/javascript-api/k6-net-grpc/stream/_index.md [hunk 2]
```diff
@@ -8,11 +8,12 @@ weight: 30
...
+| --------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you could pass [`params` for additional configurations](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-net-grpc/stream/_index.md:13]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you can pass [`params` for additional configuration options](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```

File: docs/sources/v0.49.x/javascript-api/k6-net-grpc/_index.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ The `k6/net/grpc` module provides a [gRPC](https://grpc.io/) client for Remote P
...
 | [Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response)                                          | Returned by RPC requests.                                                                                                                               |
 | [Constants](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/constants)                                        | Define constants to distinguish between [gRPC Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response) statuses.         |
-| [Stream](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                                              | Creates a new GRPC stream.                                                                                                                              |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new GRPC stream.                                                                                                                              |
```
Comment (heitortsergent) [inline docs/sources/v0.49.x/javascript-api/k6-net-grpc/_index.md:21]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new gRPC stream.                                                                                                                              |
```

File: docs/sources/v0.50.x/javascript-api/k6-net-grpc/_index.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The `k6/net/grpc` module provides a [gRPC](https://grpc.io/) client for Remote P
...
 | [Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response)                                          | Returned by RPC requests.                                                                                                                               |
 | [Constants](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/constants)                                        | Define constants to distinguish between [gRPC Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/response) statuses.         |
-| [Stream](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                                              | Creates a new GRPC stream.                                                                                                                              |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new GRPC stream.                                                                                                                              |
```
Comment (heitortsergent) [inline docs/sources/v0.50.x/javascript-api/k6-net-grpc/_index.md:21]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream)                      | Creates a new gRPC stream.                                                                                                                              |
```

File: docs/sources/v0.49.x/javascript-api/k6-net-grpc/stream/_index.md [hunk 5]
```diff
@@ -8,11 +8,12 @@ weight: 30
...
+| --------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you could pass [`params` for additional configurations](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```
Comment (heitortsergent) [inline docs/sources/v0.49.x/javascript-api/k6-net-grpc/stream/_index.md:13]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you can pass [`params` for additional configuration options](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```

File: docs/sources/v0.50.x/javascript-api/k6-net-grpc/stream/_index.md [hunk 6]
```diff
@@ -8,11 +8,12 @@ weight: 30
...
+| --------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you could pass [`params` for additional configurations](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```
Comment (heitortsergent) [inline docs/sources/v0.50.x/javascript-api/k6-net-grpc/stream/_index.md:13]:
```suggestion
| [Stream(client, url, [,params])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/grpc/stream) | Using a connected [gRPC client](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/client) and the method's URL creates a stream. Optionally, you can pass [`params` for additional configuration options](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/params) like tags, metadata, etc. |
```

**Review by heitortsergent — APPROVED** [review]
Left some small edits, other than that everything looks good! Might need to run prettier again on the files to fix the tables if you accept them. 🙇

**Comment by olegbespalov** [comment]
@heitortsergent thanks! :+1: I hope I've applied all the suggestions in a separate commit. Could you please have a final look and re-approve it? :relaxed:

**Review by heitortsergent — APPROVED** [review]
LGTM! 🚀 :shipit:

**Pull request merged**